source/elements/oneDAL/source/glossary.rst

```diff
@@ -114,11 +135,15 @@ Machine learning terms
         A special property of the :term:`data format`: the data are stored as one contiguous memory block.
 
     Data format
-        The representation of internal structure of the data: this information is enough to efficiently access them.
+        The representation of internal structure of the data.
+
+        Examples: data can be stored in array-of-structures or compressed-sparse-row format.
 
     Data type
         An integral or floating-point type which is used to physically store the data.
```

Comment:
just represent or describe the data?

Reply:
I think this term can be mentioned for both reasons. When we are talking about metadata, we mind "to describe and represent".  When we are talking about some variable which has a particular data type, we mean "to represent".

Reply:
type is used to store the data..but the data does not seem to be stored by type

Reply:
Reformulated